package.json

```diff
@@ -10,6 +10,7 @@
     "cypress-generate": "marge -o ./cypress-gen/ -f cypress-report -t 'OpenShift Console Cypress Test Results' -p 'OpenShift Cypress Test Results' --showPassed false --assetsDir ./cypress-gen/cypress/assets ./cypress-gen/cypress.json",
     "cypress-postreport": "yarn cypress-merge && yarn cypress-generate",
     "test-cypress": "node_modules/.bin/cypress open --config-file ./cypress/cypress.json --env openshift=true",
+    "test-cypress-ci": "./e2e-ops.sh && node_modules/.bin/cypress open --config-file ./cypress/cypress.json --env openshift=true",
```

Comment:
just want to confirm is naming is correct... `test-cypress-ci` make me feel like this commands gets executed in CI, but this is only needed in local...

Reply:
> Maybe `test-cypress-local`?

`local` can be also misleading as plain `test-cypress` is meant to run in local...

Following descriptive naming I think that `test-cypress-with-op-install` reveals its purpose.